### PR TITLE
Neutralize remaining instances of gendered language

### PR DESCRIPTION
**Pull request by leetfin**

### CODE REVIEW

File: doc/nyan/api_reference/reference_resistance.md [hunk 1]
```diff
@@ -317,7 +317,7 @@ Resistance to the `MakeHarvestable` effect.
 Resource spot that should be made harvestable. Effects of type `effect.discrete.make_harvestable.type.MakeHarvestable` are matched to this resistance if they store the same `ResourceSpot` object in their `resource_spot` member. Additionally, the target needs to have a `Harvestable` ability that contains the resource spot.
 
 **resist_condition**
-Condition which must he fulfilled to make the resource spot harvestable.
+Condition which they must fulfill to make the resource spot harvestable.
```
Comment (heinezen) [inline doc/nyan/api_reference/reference_resistance.md:320]: I think that's just a typo and should say "be fulfilled" :D

**Review by heinezen — CHANGES_REQUESTED** [review]
That should work. There's just one thing that you'd need to change.

Also, you need to add your info to https://github.com/SFTtech/openage/blob/master/copying.md

**Comment by leetfin** [comment]
Do I really need to be added to the authors? This seems like such a small change.

**Review by TheJJ — APPROVED** [review]

**Comment by TheJJ** [comment]
it's fine, it's not a code contribution :)

**Pull request merged**